// chinese-calendar/lib/constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.HOLIDAYS_TYPE = {
    NEW_YEARS_DAY: ["New Year's Day", '元旦', 1],
    SPRING_FESTIVAL: ['Spring Festival', '春节', 3],
    TOMB_SWEEPING_DAY: ['Tomb-sweeping Day', '清明', 1],
    LABOUR_DAY: ['Labour Day', '劳动节', 1],
    DRAGON_BOAT_FESTIVAL: ['Dragon Boat Festival', '端午', 1],
    MID_AUTUMN_FESTIVAL: ['Mid-autumn Festival', '中秋', 1],
    NATIONAL_DAY: ['National Day', '国庆节', 3],
};
exports.Holidays = new Map([
    ['2020-01-01', exports.HOLIDAYS_TYPE.NEW_YEARS_DAY],
    ['2020-01-24', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-25', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-26', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-27', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-28', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-29', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-30', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-01-31', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2020-05-01', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2020-05-02', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2020-05-03', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2020-05-04', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2020-05-05', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2020-06-25', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2020-06-26', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2020-06-27', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2020-10-01', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-02', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-03', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-04', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-05', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-06', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-07', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2020-10-08', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-01-01', exports.HOLIDAYS_TYPE.NEW_YEARS_DAY],
    ['2021-01-02', exports.HOLIDAYS_TYPE.NEW_YEARS_DAY],
    ['2021-01-03', exports.HOLIDAYS_TYPE.NEW_YEARS_DAY],
    ['2021-02-11', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-12', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-13', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-14', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-15', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-16', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-02-17', exports.HOLIDAYS_TYPE.SPRING_FESTIVAL],
    ['2021-04-03', exports.HOLIDAYS_TYPE.TOMB_SWEEPING_DAY],
    ['2021-04-04', exports.HOLIDAYS_TYPE.TOMB_SWEEPING_DAY],
    ['2021-04-05', exports.HOLIDAYS_TYPE.TOMB_SWEEPING_DAY],
    ['2021-05-01', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2021-05-02', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2021-05-03', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2021-05-04', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2021-05-05', exports.HOLIDAYS_TYPE.LABOUR_DAY],
    ['2021-06-12', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2021-06-13', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2021-06-14', exports.HOLIDAYS_TYPE.DRAGON_BOAT_FESTIVAL],
    ['2021-09-19', exports.HOLIDAYS_TYPE.MID_AUTUMN_FESTIVAL],
    ['2021-09-20', exports.HOLIDAYS_TYPE.MID_AUTUMN_FESTIVAL],
    ['2021-09-21', exports.HOLIDAYS_TYPE.MID_AUTUMN_FESTIVAL],
    ['2021-10-01', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-02', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-03', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-04', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-05', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-06', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
    ['2021-10-07', exports.HOLIDAYS_TYPE.NATIONAL_DAY],
]);
